ci/taos/plugins-good/pr-prebuild-doxygen-tag.sh

```diff
@@ -29,27 +29,29 @@
 ##
 # @brief [MODULE] ${BOT_NAME}/pr-prebuild-doxygen-tag
 function pr-prebuild-doxygen-tag(){
-    echo "########################################################################################"
-    echo "[MODULE] ${BOT_NAME}/pr-prebuild-doxygen-tag: Check if source code includes required Doxygen tags for Doxygen documentation."
+    report_path=../report/doxygen_tag.txt
+
+    echo "########################################################################################" >> $report_path
+    echo "[MODULE] ${BOT_NAME}/pr-prebuild-doxygen-tag: Check if source code includes required Doxygen tags for Doxygen documentation." >> $report_path
```

Comment:
Q: Previously, this log has been printed to `pr-prebuild-group.txt`. Does this condition still hold?

Reply:
No this won't print to the prebuild-group anymore. I found that others making their own report and not passing report contents to the `pr-prebuild-group.txt` except for the doxygen-tags, so made this for consistency.

Reply:
I think doxygen-specific report is fine. But, this message can be regarded as a common report.
I can still see similar messages in http://nnstreamer.mooo.com/TAOS-CI/ci/repo-workers/pr-checker/719-202104221338500.35172891616821-bc99590a28b60afffc1b5b8e1b57f2910d18e123/pr-prebuild-group.txt for other prebuild checkers.

I think you may have to consider the uniformity.

Reply:
Understood. I have change those two lines and the status report back to the group :)